src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -207,12 +216,12 @@ export class CreateScenariosComponent implements OnInit {
         }
         // setting treatment question
         if (scenario.configuration.treatment_question) {
-          this.prioritiesComponent.setFormData(
+          this.prioritiesComponent?.setFormData(
```

Comment:
Same here- I don't think we should add optional chaining where we actually want to set stuff